src/pages/jeonyongmin/Main/components/SubMain/Feed/Feed.js

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import Comment from './components/Comment';
+import './Feed.scss';
+
+class Feed extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      inputValue: '',
+      buttonColor: false,
+      commentList: [],
+    };
+    this.inputLocation = React.createRef();
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/jeonyongmin/commentData.json', {
+      method: 'GET',
+    })
+      .then(result => result.json())
+      .then(data => {
+        this.setState({
+          commentList: data[this.props.number - 1],
+        });
+      });
```

Comment:
어떤 data인지 좀 더 구체적으로 써주시면 좋을 것 같아요!! 
```suggestion
      .then(commentData => {
        this.setState({
          commentList: commentData[this.props.number - 1],
        });
      });
```

Reply:
넵 확인했습니다!